extensions/amp-story/1.0/test/test-amp-story-bookend.js

```diff
@@ -139,7 +138,7 @@ describes.fakeWin('amp-story-bookend', {win: {location}, amp: true}, (env) => {
     requestService = new AmpStoryRequestService(win, storyElem);
     env.sandbox.stub(Services, 'storyRequestService').returns(requestService);
 
-    const localizationService = new LocalizationService(win);
+    const localizationService = Services.localizationServiceForDoc(env.ampdoc);
```

Comment:
Here and in all the other test files, what these new lines are doing are:
1. Get the registered service
2. Register the service with what was already registered

I think we should be able to get rid of L141-L144 :))

Reply:
Woops, I missed they were being registered. Good catch.